src/lib/icons.js

```diff
@@ -94,4 +116,26 @@ export const mIcons = {
   risecircle: IcLrgRise,
   growthcircle: IcLrgGrowth,
   enterprisecircle: IcLrgEnterprise,
+  start: IcColorStart,
+  cart: IcColorCart,
+  checkout: IcColorCheckout,
+  orders: IcColorOrders,
+  products: IcColorProducts,
+  categories: IcColorCategories,
+  backend: IcLrgBackend,
+  business: IcLrgBusiness,
+  'color-checkout': IcLrgCheckout,
```

Comment:
Should this be using `IcColorCheckout` instead of `IcLrgCheckout`?

Reply:
ah yea thats confusing, it's supposed to use IcLrgCheckout, will change label to 'checkout-lrg'!

Reply:
`lg` or `large` is preferable. `lg` because that's what we use with all the tailwind stuff, or `large` because it's the actual word.